e2e/runJest.js

```diff
@@ -30,6 +31,16 @@ export default function runJest(
   dir: string,
   args?: Array<string>,
   options: RunJestOptions = {},
+) {
+  return normalizeResult(spawnJest(dir, args, options), options);
+}
+
+// Spawns Jest and returns either a Promise (if spawnAsync is true) or the completed child process
+function spawnJest(
```

Comment:
Pretty nice refactor to avoid the code duplication btw, regardless of the actual fix 👍

Reply:
Yeah - I was gonna circle back to this when I wrote it, but never got around to it... Nice and clean now :)